.azure/scripts/metadata_validator.py

```diff
@@ -10,12 +10,13 @@
 VALID_COMPONENT_TYPES = [
     "service", "webhook", "manual-task", "scheduled-task", 
     "event-triggered", "event-handler", "test-runner", "many",
-    "web-application"
+    "web-application", "git-proxy"
 ]
 
 VALID_BUILD_PACKS = [
     "ballerina", "wso2-mi", "go", "java", "php", "python", "nodejs", "ruby",
-    "vue", "many", "postman", "react", "docker", "angular", "dotnet", "prism-mock"
+    "vue", "many", "postman", "react", "docker", "angular", "dotnet", "prism-mock",
+    "none" 
```

Comment:
Please add this to the `buildPackNameMap` in choreo-console.
https://github.com/wso2-enterprise/choreo-console/blob/7351666de3727a38fda71d6a551bdb4a20b7f758/src/hooks/samples.tsx#L38C9-L38C25

Reply:
Addressed : https://github.com/wso2-enterprise/choreo-console/pull/9704